compiler-rt/test/builtins/Unit/compiler_rt_fmaxl_test.c

```diff
@@ -6,7 +6,9 @@
 #include <stdio.h>
 #include "fp_lib.h"
 
-#if defined(CRT_HAS_TF_MODE)
+// TODO: This test currently only works if long double is the same as fp_t.
+// On x86_64 we get undefined references to __trunctfxf2 and __extendxftf2
```

Comment:
jyfi - I'm working on __trunctfxf2 and __extendxftf2 (https://github.com/llvm/llvm-project/pull/66918)

Reply:
I can drop this part of the diff if your change lands first.

Reply:
It's merged now. You can drop this part

Reply:
Seems like the tests fail if I enable them. Will need to debug this.

Reply:
Do we have more insights on these failures? extend and trunc functions that got introduced in #66918 may benefit from more testing. So there's a chance that these failures mean some bugs in this newly introduced extend/trunc implementations.

Reply:
There was a mismatch in the upper bits of the extended value in at least one test case, I will try to get the log later today.

Reply:
Added ifdefs to skip the tests that can't work with non-IEEE 128 tf types